librsvg: build fails

```
libtool:   error: Could not determine the host path corresponding to
libtool:   error:   'C:/msys64/mingw64/lib:C:/msys64/mingw64/bin:/home/user/M/mingw-w64-librsvg/src/build-x86_64-w64-mingw32/.libs:/mingw64/lib:/mingw64/bin'
libtool:   error: Continuing, but uninstalled executables may not work.
  GICOMP   Rsvg-2.0.gir
 VAPIGEN librsvg-2.0.vapi
GdkPixbuf-2.0.gir:6448.55-6448.55: error: The symbol `GModule' could not be found
make[2]: *** [C:/msys64/mingw64/share/vala/Makefile.vapigen:61: librsvg-2.0.vapi] Error 1
make[2]: *** Waiting for unfinished jobs....
```

--- a/mingw-w64-librsvg/0006-fix-vapigen-deps.patch
+++ b/mingw-w64-librsvg/0006-fix-vapigen-deps.patch
@@ -1,0 +1,11 @@
+--- librsvg-2.50.3/Makefile.am.orig	2021-07-10 17:47:36.531222200 +0200
++++ librsvg-2.50.3/Makefile.am	2021-07-10 17:47:38.774497600 +0200
+@@ -340,7 +340,7 @@
+ 
+ VAPIGEN_VAPIS = librsvg-$(RSVG_API_VERSION).vapi
+ 
+-librsvg_@RSVG_API_VERSION_U@_vapi_DEPS = gio-2.0 cairo
++librsvg_@RSVG_API_VERSION_U@_vapi_DEPS = gio-2.0 cairo gdk-pixbuf-2.0
+ librsvg_@RSVG_API_VERSION_U@_vapi_METADATADIRS = $(srcdir)
+ librsvg_@RSVG_API_VERSION_U@_vapi_FILES = Rsvg-$(RSVG_API_VERSION).gir $(srcdir)/Rsvg-$(RSVG_API_VERSION)-custom.vala
+ 

--- a/mingw-w64-librsvg/PKGBUILD
+++ b/mingw-w64-librsvg/PKGBUILD
@@ -4,7 +4,7 @@ _realname=librsvg
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=2.50.3
-pkgrel=1
+pkgrel=3
 pkgdesc="SVG rendering library (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
@@ -15,7 +15,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-gobject-introspection"
              "${MINGW_PACKAGE_PREFIX}-vala"
              "${MINGW_PACKAGE_PREFIX}-rust>=1.26"
-             "setconf")
+             "${MINGW_PACKAGE_PREFIX}-gtk-doc")
 depends=("${MINGW_PACKAGE_PREFIX}-gdk-pixbuf2"
          "${MINGW_PACKAGE_PREFIX}-pango"
          "${MINGW_PACKAGE_PREFIX}-cairo"
@@ -23,13 +23,18 @@ depends=("${MINGW_PACKAGE_PREFIX}-gdk-pixbuf2"
 optdepends=("${MINGW_PACKAGE_PREFIX}-gtk3: for rsvg-view-3")
 options=('staticlibs' 'strip')
 source=("https://download.gnome.org/sources/librsvg/${pkgver%.*}/${_realname}-${pkgver}.tar.xz"
-        "0005-hack-unixy-paths.patch")
+        "0005-hack-unixy-paths.patch"
+        "0006-fix-vapigen-deps.patch")
 sha256sums=('a4298a98e3a95fdd73c858c17d4dd018525fb09dbb13bbd668a0c2243989e958'
-            'b23b094c0cb65fcbbbb952350448de6f3430b30f273e05acdbf7a56d634212dc')
+            'b23b094c0cb65fcbbbb952350448de6f3430b30f273e05acdbf7a56d634212dc'
+            '8ac94e8b6b67c57d04f547e09a50b50a5722365b18f4ed76fb7cd0a9bc27d9c6')
 
 prepare() {
   cd "${srcdir}/${_realname}-${pkgver}"
   patch -p1 -i "${srcdir}/0005-hack-unixy-paths.patch"
+
+  # https://gitlab.gnome.org/GNOME/vala/-/issues/1199
+  patch -p1 -i "${srcdir}/0006-fix-vapigen-deps.patch"
 
   autoreconf -fiv
 }

